packages/universal-components/src/RangeDatePicker/NightsInDestination.js

```diff
@@ -0,0 +1,48 @@
+// @flow
+
+import * as React from 'react';
+import { View } from 'react-native';
+import { defaultTokens } from '@kiwicom/orbit-design-tokens';
+
+import { Slider } from '../Slider';
+import { StyleSheet } from '../PlatformStyleSheet';
+
+type Props = {|
+  +onNightsInDestinationChange?: ?(Array<number>) => void,
+  +nightsInDestination?: ?$ReadOnlyArray<number>,
+|};
+
+export default class NightsInDestination extends React.Component<Props> {
+  render() {
+    return (
+      this.props.nightsInDestination != null &&
+      this.props.onNightsInDestinationChange != null && (
+        <View style={styles.container}>
+          <View style={styles.container} />
+          <View style={styles.slider}>
+            <Slider
```

Comment:
This is probably `Slider` problem. So it definitely doesn't need to be part of this PR.
But currently I cannot select values with 1 step distance (for example `11-12` range).
Can we at least create issue for it?

![limit-problem](https://user-images.githubusercontent.com/2660330/57614304-2728f900-7579-11e9-8585-d31667ddcd70.gif)

Another thing which can be saved for separate issue/PR. Selecting just single value instead of range should be possible (like on Kiwi.com).

![single-val](https://user-images.githubusercontent.com/2660330/57614305-27c18f80-7579-11e9-8564-53cd9d02760d.gif)

Reply:
#762  
#807